docs/source/community/rfc/rfc-4.rst

```diff
@@ -0,0 +1,1015 @@
+  geographic horizontal shift grids, or geographic to vertical shift grids), but
+  for vertical to vertical CRS adjustment, this will be the geographic CRS to
+  which the grid is referenced. In some very rare cases of geographic to vertical
+  shift grids, the interpolation CRS might be a geographic CRS that is not the
+  same as the source CRS (into which ellipsoidal height are expressed). The only
+  instance we have in mind is for the EPSG:7001 "ETRS89 to NAP height (1)" transformation
+  using the naptrans2008 VDatum-grid which is referenced to Amersfoort EPSG:4289
+  instead of ETRS89...
+
+  On the reading side, PROJ will ignore that information:
+  the CRS is already stored in the source_crs or interpolation_crs column of the
+  grid_transformation table.
+
+  For geographic to vertical shift files (geoid models), the GeoTIFF 1.1
+  convention will be used to store the value of the `VerticalGeoKey
+  <http://docs.opengeospatial.org/is/19-008r4/19-008r4.html#_requirements_class_verticalgeokey>`_
+  So a geoid model that apply to WGS 84 EPSG:4979 will have GeodeticCRSGeoKey = 4326
+  and VerticalGeoKey = 4979.
+
+- Files hosted on the CDN will use the GeoTIFF defined `ModelTiepointTag and ModelPixelScaleTag
+  <http://docs.opengeospatial.org/is/19-008r4/19-008r4.html#_raster_to_model_coordinate_transformation_requirements>`_ TIFF tags
+  to store the coordinates of the upper-left pixel and the resolution of the pixels.
+  On the reading side, they will be required and ModelTransformationTag will be ignored.
+
+  .. note::
+
+    Regarding anti-meridian handling, a variety of possibilities exist.
+    We do not attempt to standardize this and filesh hosted on the CDN will use
+    a georeferencing close to the original data producer.
+    For example, NOAA vertical grids that apply to Conterminous USA might even have a top-left
+    longitude beyond 180 (for consistency with Alaska grids, whose origin is < 180)
+    Anti-meridian handling in PROJ has probably issues. This RFC does not attempt
+    to address them in particular, as they are believed to be orthogonal to the
+    topics it covers, and being mostly implementation issues.
+
+- Files hosted on the CDN will use the `GTRasterTypeGeoKey
+  <http://docs.opengeospatial.org/is/19-008r4/19-008r4.html#_requirements_class_gtrastertypegeokey>`_
+  = PixelIsPoint convention.
+  This is the convention used by most existing grid formats currently. Note that GDAL
+  typically use a PixelIsArea convention (but can handle both conventions), so the
+  georeferencing it displays when opening a .gsb or .gtx file appears to have a 
+  half-pixel shift regarding to the coordinates stored in the original grid file. On
+  the reading side, PROJ will accept both conventions (for equivalent georeferencing,
+  the value of the origin in a PixelIsArea convention is shifted by a half-pixel
+  towards the upper-left direction). Unspecified behaviour if this GeoKey is absent.
```

Comment:
no, that wasn't my intention. In GeoTIFF PixelIsPoint vs PixelIsArea is mostly a matter of taste. But given the convention of sources we have, PixelIsPoint is better because there's no need to insert a half-pixel shift. It is thus easier to check that we kept the georeferencing of the source preserved

Reply:
Actually, I've proposed later in the doc a `recommended_interpolation_method` (name coming from a GGXF doc actually) that could be used to specify the interpolation method. But I don't particularly plan on changing how we interpolate currently (bilinear AFAIK)

Reply:
Got it. No, unless some particular use case shows up, then I would say the bilinear interpolation is just fine. We can deal with if and when needed.

Reply:
> unless some particular use case shows up

The GDA94 to AHD height transformation in EPSG mentions "Uses AusGeoid09 model which uses bi-cubic interpolation; bi-linear interpolation of the grid file will give results agreeing to within 1cm 99.97% of the time". But I don't feel I have "compelling" reasons to implement that right now :-)

Reply:
I'm sure the Aussies will open a pull request with an implementation if it is of use to them :-)